packages/caver-transaction/src/transactionTypes/abstractFeeDelegatedTransaction.js

```diff
@@ -0,0 +1,221 @@
+
+        this.appendFeePayerSignatures(sigs)
+
+        return this
+    }
+
+    /**
+     * Appends feePayerSignatures to the transaction.
+     *
+     * @param {Array.<string>|Array.<Array.<string>>} sig - An array of feePayerSignatures to append to the transaction.
+     *                                                      One feePayerSignature can be defined in the form of a one-dimensional array or two-dimensional array,
+     *                                                      and more than one feePayerSignatures should be defined in the form of a two-dimensional array.
+     */
+    appendFeePayerSignatures(sig) {
+        if (!Array.isArray(sig[0])) sig = [sig]
+        this.feePayerSignatures = this.feePayerSignatures.concat(sig)
+    }
+
+    /**
+     * Combines signatures and feePayerSignatures to the transaction from RLP-encoded transaction strings and returns a single transaction with all signatures combined.
+     * When combining the signatures into a transaction instance,
+     * an error is thrown if the decoded transaction contains different value except signatures.
+     *
+     * @param {Array.<string>} rlpEncodedTxs - An array of RLP-encoded transaction strings.
+     * @return {string}
+     */
+    combineSignatures(rlpEncodedTxs) {
+        if (!_.isArray(rlpEncodedTxs)) throw new Error(`The parameter must be an array of RLP encoded transaction strings.`)
+        formatters.inputTransactionFormatter(this)
+
+        // If the signatures are empty, there may be an undefined member variable.
+        // In this case, the empty information is filled with the decoded result.
+        let fillVariables = false
+        if (this.signatures.length === 0 && this.feePayerSignatures.length === 0) fillVariables = true
+
+        for (const encoded of rlpEncodedTxs) {
+            const type = typeDetectionFromRLPEncoding(encoded)
+            if (this.type !== type) throw new Error(`Transaction type mismatch: Signatures from different transactions cannot be combined.`)
+
+            const decoded = this.constructor.decode(encoded)
+
+            // Signatures can only be combined for the same transaction.
+            // Therefore, compare whether the decoded transaction is the same as this.
+            for (const k in decoded) {
+                if (k === '_signatures' || k === '_+feePayerSignatures') continue
+                if (k === '_feePayer') {
+                    if ((decoded[k] !== '0x' || this[k] === '0x') && fillVariables) this[k] = decoded[k]
+                }
+
+                if (this[k] === undefined && fillVariables) this[k] = decoded[k]
+
+                if (this[k] !== decoded[k]) {
+                    throw new Error(`Transactions containing different information cannot be combined.`)
+                }
+            }
+
+            this.appendSignatures(decoded.signatures)
+            this.appendFeePayerSignatures(decoded.feePayerSignatures)
```

Comment:
Is it okay if we have signatures == [] and feePayerSignatures == [v,r,s]?

Reply:
I think that will be fine, because signatures will be refined in setter.
In setter refineSignatures will be called.
I forgot to add transactionHelper file ;-; so i added that file in this PR also !